geom/dcel_overlay.go

```diff
@@ -90,10 +41,17 @@ func (d *doublyConnectedEdgeList) fixVertices() {
 
 func (d *doublyConnectedEdgeList) fixVertex(v *vertexRecord) {
 	// Sort the edges radially.
-	if len(v.incidents) >= 3 {
-		sort.Slice(v.incidents, func(i, j int) bool {
-			ei := v.incidents[i]
-			ej := v.incidents[j]
+	incidents := make([]*halfEdgeRecord, 0, len(v.incidents))
+	for e := range v.incidents {
+		incidents = append(incidents, e)
+	}
+	if len(incidents) >= 3 {
+		// TODO: consider using a solution like
```

Comment:
I've created a ticket to address this here: https://github.com/peterstace/simplefeatures/issues/474